### ISSUE TITLE
KeeAgent not starting on Linux

### ISSUE THREAD

**Issue by pahofmann** [dlech/KeeAgent#123]
I get the following error when compilation is finished after the starting keepass:
```
 KeeAgent failed to load:
 Failed to start Unix Agent: Object reference not set to instance of an object
```
Any ideas whats causing this?

**Comment by dlech** [comment]
I have just added Agent mode support for Linux, so you have found a bug with this. Unfortunate, this is not enough information to find out exactly what the problem is, but I will try a clean install and see if I can reproduce the error.

To get your KeePass working again, you will probably have to manually edit your KeePass.config.xml file to set it to client mode. On Linux, this file is usually found at `~/.config/KeePass/` (or possibly in the same folder as `KeePass.exe`. Open it in a text editor and look for "KeeAgent.AgentMode":

		<Item>
			<Key>KeeAgent.AgentMode</Key>
			<Value>Auto</Value>
		</Item>


change the Value to "Client"

    <Value>Client</Value>

KeePass/KeeAgent should work as before.

**Comment by pahofmann** [comment]
I've edited the line now I get the following error when unlocking my database:
```
KeeAgent: Error while loading key from entry xxxxxxx
Could not add key because no SSh agent was found.
Please make sure your SSH agent is running (e.g. pagent)
```
Is there something else I need to configure to get it working with OpenSSH?

If I can provide Debug information to help just let me know what you need.

**Comment by dlech** [comment]
Was this working before? Do you have `ssh-agent` running? (you can test this by running `ssh-add -l` in a terminal).

**Comment by pahofmann** [comment]
No havent used it before under linux, so it wasnt working before.`
`ssh-agent`  is running but `ssh-add` states `Could not open a connection to your authentication agent.` so there seems to be a problem with the agent.

**Comment by dlech** [comment]
Did you start `ssh-agent` manually? It needs to be started by your session manager so that it can be seen by all processes. What is `echo $SSH_AUTH_SOCK`? I'm gessing not anything since `ssh-add` didn't work.

What Linux distro and desktop environment are you using?

**Comment by pahofmann** [comment]
I'm using gnome on archlinux.
I disabled the ssh-component of the gnome-keyring as mentioned in the first step of the installation page. That seems to have caused the issue. After reverting the changes ssh-agent is starting normal and everything works fine.

**Comment by dlech** [comment]
This is fixed in v0.7.9.
